src/Diff.re

```diff
@@ -15,50 +15,86 @@ module MakeDiff = (IO1: ImageIO.ImageIO, IO2: ImageIO.ImageIO) => {
         ~threshold=0.1,
         (),
       ) => {
-    let diffCount = ref(0);
+    let diffPixels = ref([]);
     let maxDelta = maxYIQPossibleDelta *. threshold ** 2.;
     let diffOutput = outputDiffMask ? IO1.makeSameAsLayout(base) : base;
 
     let countDifference = (x, y) => {
-      diffCount := diffCount^ + 1;
-      diffOutput |> IO1.setImgColor(x, y, diffPixel);
+      diffPixels := [(x, y), ...diffPixels^];
```

Comment:
I updated this, to not immediately draw on the image we are currently of diffing.

With `outputDiffMask` set to true, this wouldn't be a problem, as we are drawing on a new (transparent) image. When it is set to false however, we are drawing on (and modifying) the original image, to which we might come back to check for antialiasing.